Android/app/src/main/res/menu/share_contact_menu.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+    <item
+        android:id="@+id/share_contact"
+        android:title="share"
```

Comment:
add "share" string to strings.xml file, and use that here.
Don't add constant strings directly in xml file